plugins/github-deployments/src/index.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export {
+  githubDeploymentsPlugin as plugin,
```

Comment:
Last thing from me - not sure we should really remove the specificity here, as we end up having to always rename them where we use them?

I think I like `githubDeploymentsPlugin` and `isGithubDeploymentsAvailable`. What do you think?

Reply:
yes, this should definitely be keeping its original name, i started commenting in the app and docs code above to that effect as well but i'll just leave my agreeing in here instead